package.json

```diff
@@ -6,7 +6,7 @@
     "To turn spell checking off, remove it from the chromium-args in this file",
```

Comment:
Took a lot of trial and error, but at this point I think I've got everything working except for this error:

![error](https://user-images.githubusercontent.com/4629794/123502219-b938d200-d618-11eb-9690-ad1bd4310ea1.png)

The main difficulty in trying to fix all these issues came from not being able to understand the TS code in the documentation. If it was written in English or JavaScript, it would have saved me a lot of time. As it stands, any TS might as well not even exists on there as I wasted time trying to understand it and ultimately couldn't. It was extremely frustrating and felt like a Python library putting it's documentation in C or something.

Several bugs were fixed just by updating from RC6 to RC7.

Reply:
Hmm this looks like something to do with the Babel x TypeScript. I have no seen this one before 🤔

Reply:
TS should not be in use in this project